boards/beagle/pocketbeagle_2/support/openocd.cfg

```diff
@@ -0,0 +1,8 @@
+# BeagleBoard.org PocketBeagle 2
+#
+# Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Requires External XDS110
+source [find board/ti_am625evm.cfg]
```

Comment:
OK, I guess this makes it clear you are using JTAG and not native memory OpenOCD. Can you support both?

Reply:
I will add it if I get it working. However, I will first work on remoteproc console and testing all the different peripherals like GPIO, I2C etc.